### DISCUSSION TITLE
How to deal with high input lag?

### DISCUSSION THREAD

**Discussion by ssl33pl3ss** [cssbattle/forum#175] [Q&A]
When I enter 1 line of code, I need to wait about 5-10 seconds for it to appear. Is it problem with my PC, or with website?

**Comment by RazvanManolache** [comment]
Probably with your pc. Check if you have some extensions that interfere. Try a different browser to check first.

**Reply by ssl33pl3ss** [reply]
clean browser with no extensions didn't help

**Reply by chinchang** [reply]
What about trying a different browser?

**Reply by ssl33pl3ss** [reply]
same

**Comment by rafehimself** [comment]
i would first do this: 
![image](https://user-images.githubusercontent.com/58576132/141934207-e76b091c-79a4-49e1-9053-cc3e01278ab7.png)

open command prompt, and write ping cssbattle.dev -t
see what delay do you have

**Reply by ssl33pl3ss** [reply]
it was first thing i did and average was 63ms
